docs/tools/hardhat/hardhat-zksync-verify.md

```diff
@@ -142,4 +151,12 @@ const verificationId = await hre.run("verify:verify", {
 This task returns a verification id if the request was successfully sent.<br/>
 You can use this id to check the status of your verification request as described in the section above.
 
-If the request was NOT successful, the return value is `-1`.
+If you are using encoded constructor args, `constructorArguments` parameter should be a non-array value starting with `0x`.
```

Comment:
This is a breaking change isn't it? Up until now constructorArguments just accepted the params in an array.